pulseworker/main.py

```diff
@@ -0,0 +1,116 @@
+import datetime
+import functools
+import logging
+import os
+import sys
+import json
+
+import requests
+from kinto_http import cli_utils
+from mozillapulse.consumers import NormalizedBuildConsumer
+
+
+DEFAULT_SERVER = "https://kinto-ota.dev.mozaws.net/v1"
+DEFAULT_BUCKET = "build-hub"
+DEFAULT_COLLECTION = "builds"
+PULSE_TOPIC = "build.#"
+PULSEGUARDIAN_USER = os.getenv("PULSEGUARDIAN_USER", "")
+PULSEGUARDIAN_PASSWORD = os.getenv("PULSEGUARDIAN_PASSWORD", "")
+
+
+logger = logging.getLogger(__name__)
+
+
+def epoch2iso(timestamp):
+    dt = datetime.datetime.utcfromtimestamp(timestamp)
+    # XXX: Python 3
+    if sys.version_info >= (3, 2):
+        dt = dt.replace(tzinfo=datetime.timezone.utc)
+    return dt.isoformat()
+
+def update_download_info(client, record):
+    rid = record["data"]["id"]
+    dlinfo = dict(record["data"]["download"])
+
+    resp = requests.head(dlinfo["url"])
+    dlinfo["size"] = int(resp.headers["Content-Length"])
+    dlinfo["mimetype"] = resp.headers["Content-Type"]
+
+    # XXX: use JSON-merge header.
+    client.patch_record({"download": dlinfo}, id=rid)
+
+
+def pulse2kinto(body, msg, client):
+    # https://github.com/mozilla/pulsetranslator#routing-keys
+    buildinfo = body["payload"]
+
+    skip = (
+        buildinfo["tree"] in ("autoland", "try") or
+        buildinfo["buildtype"] in ("pgo",) or
+        buildinfo["tree"] in ("mozilla-inbound",))
```

Comment:
are you expecting some more "skip" in the future for the buildtype and the tree? It feels a bit weird to have this `in` with a single-element list